lib/Dredd.js

```diff
@@ -47,8 +45,10 @@ class Dredd {
       end: 0,
       duration: 0,
     };
+    this.configuration.files = [];
```

Comment:
Will be renamed to `this.files` as the next step, as it is something completely local to the `Dredd` class 🙄